src/components/Fabric/Fabric.tsx

```diff
@@ -49,6 +44,11 @@ export class Fabric extends React.Component<React.HTMLProps<HTMLDivElement>, IFa
   public componentDidMount() {
     this._events.on(document.body, 'mousedown', this._onMouseDown, true);
     this._events.on(document.body, 'keydown', this._onKeyDown, true);
+
+    // Ensure that the HTML element has a dir specified. This helps to ensure RTL/LTR macros in css for all components will work.
+    if (document && document.documentElement && !document.documentElement.getAttribute('dir')) {
+      document.documentElement.setAttribute('dir', 'ltr');
+    }
```

Comment:
@annapablo Since this is in Fabric I believe you should take this section of code out entirely and make sure the Fabric component is used somewhere.